### DISCUSSION TITLE
HomematicIP Lokal

### DISCUSSION THREAD

**Discussion by trytogetme85** [community/community#31793] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi und Danke für dein HASS-Addon.
Das Problem was ich habe ist, dass ich von den Heizkörper-Thermostaten die Temperatur nicht ausgelesen bekomme.
Das normale "Thermostat" Ding von HomeAssistant zeigt die Temp aber an. Im Gerät selbst steht, die Entität wäre nicht verfügbar...weißt du woran das liegt?
https://imgur.com/a/UqbqR4W

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're looking for help with a software hosted on GitHub, though I can't guess which, there are a lot of repositories with "HomematicIP" in the name. In that case you should search for the right repository and open an issue there so the developers see it. :slightly_smiling_face:

**Reply by trytogetme85** [reply]
Yes, my fault posting this here :(

**Reply by airtower-luna** [reply]
I hope you find the right repository soon!
